torch/distributed/tensor/_api.py

```diff
@@ -606,6 +606,16 @@ def __create_write_items__(self, fqn: str, object: Any):
             raise RuntimeError("Unsupported tensor type!")
 
     def __create_chunk_list__(self):
```

Comment:
Hmm, dunder functions usually has some special meaning in Python runtime. What is it decorated like that?

Reply:
I am wondering what's the special meaning refers to? iirc some dunder methods do operator overload, while others retrieves object information. This method is mainly retrieves the chunk storage metadata for distributed checkpoint purpose.